### ISSUE TITLE
Add Igbinary support for PhpRedis

### ISSUE THREAD

**Issue by ParhamG** [cytopia/devilbox#764]
#### ISSUE TYPE

 - Feature request

#### SUMMARY
Can we have the PHP Redis extension compiled with Igbinary support so that igbinary can be used as a serializer with Redis?

**Comment by cytopia** [comment]
@ParhamG is baking here: https://github.com/devilbox/docker-php-fpm/pull/188

**Comment by ParhamG** [comment]
@cytopia Thanks for the quick update!

I am looking at the update instructions [here](https://devilbox.readthedocs.io/en/latest/maintenance/update-the-devilbox.html#update-one-docker-image), and I am just a little bit confused. Looking at `docker-compose.yml` it is set to use a release specific tag for the php image (e.g. `-work-0.112`) and my understanding is that simply puling the same used image will not work here, right? And pulling the latest (`-work`) would need updating `docker-compose.yml` or overriding the `image` attribute for the `php` container in `docker-compose.override.yml` to:
```
  php:
    image: devilbox/php-fpm:${PHP_SERVER}-work
```
Am I missing something or is this actually how we are supposed to update to use the latest image?

I did update my `docker-compose.override.yml` to use the latest image and that worked for me after pulling `devilbox/php-fpm:7.4-work`

**Comment by llaville** [comment]
@ParhamG I've already noticed the same issue with docker images tags. This is a problem with Travis. Cytopia will fix it soon 😉

**Comment by cytopia** [comment]
The latest tag `0.121` is currently building by Travis: https://travis-ci.org/github/devilbox/docker-php-fpm/builds/748746927

You can use them immediately, once Travis is finished (by dirtily adjusting `docker-compose.yml` :smiling_imp: ) or wait until a new Devilbox version has been released

**Comment by ParhamG** [comment]
Got it, so it is still baking:) and once done it will be available on the latest release tag and since Devilbox is not updated as often as the images are you'd need to adjust `docker-compose.yml` to use the latest image.

But just curious would `devilbox/php-fpm:7.4-work` also point to the latest available build/release? any harm in using that in my override? that seemed to work for me and I have Igbinary available now although 0.121 is still building.

**Comment by llaville** [comment]
@ParhamG 

When I want to test new php-fpm release tag (as 0.121) before new Devilbox release (as 1.8.3), I do not touch my `docker-compose.yml`.
I create a `docker-compose.override.yml` with at least following content:

```
services:

  php:
    image: devilbox/php-fpm:${PHP_SERVER}-work-0.121
```
And If there is another release later, I will only change the 0.121 suffix. 
I run the `docker-compose pull` command to download the image from docker hub. That's all !

**Comment by llaville** [comment]
@cytopia LGTM for new feature 

```
devilbox@php-7.2.34 in /shared/backups/bartlett/php-compatinfo-db $ php --ri redis

redis

Redis Support => enabled
Redis Version => 5.3.2
Redis Sentinel Version => 0.1
Git revision => $Id: d2c927ba85b85a279cec5810e99723d14309a828 $
Available serializers => php, json, igbinary, msgpack
```

**Comment by cytopia** [comment]
> Devilbox is not updated as often as the images are

FYI: This is only due to the fact, that sometimes the images are broken already a few hours after release, they are not fully working with the current Devilbox stack or I'm already on adding new features to the images themselves and then just want to combine this all in one Devilbox release.

**Comment by ParhamG** [comment]
@cytopia I understand, it totally makes sense for Devilbox to have a separate release schedule. I am all set, thank you! And feel free to close this issue.

**Comment by cytopia** [comment]
@ParhamG it is now implemented in release `v1.9.0`
